docs/main/02-intro.md

```diff
@@ -1,71 +1,131 @@
+Developers lose productivity when forced to wait on manual processes or tickets.
+Kratix lets you offer everything as-a-service via clear APIs, which can easily
+be exposed through developer portals like Backstage. This enables developers to
+instantly self-service resources, significantly accelerating software delivery.
 
-And with Kratix's built-in [Fleet Management](./03-reference/11-promises/03-updates.md) capabilities, you can maintain and upgrade all your resources with a single click.
+### 🔒 Be Safer
 
-![Kratix Architecture](/img/kratix-architecture.svg)
+Inconsistent application of critical processes creates risks. Kratix helps you
+embed essential business rules—such as security, compliance, and
+governance—directly into automated workflows, ensuring reliable enforcement
+every time.
 
-## Who is Kratix for
+### ⚙️ Increase Efficiency
 
-### Platform Engineers
-Teams responsible for building and maintaining platforms often struggle to meet the growing expectations of application teams and organisations. Delivering the right services quickly, safely, and consistently is already challenging—adding Day 2 operations like upgrades and continuous improvements can lead to burnout.
+Manual management of fragmented infrastructure results in operational overhead.
+Kratix simplifies operations by managing platform resources as a unified,
+automated fleet, reducing complexity and scaling effortlessly as your
+organisation grows.
+
+:::info
+
+Want to chat to us about how Kratix can help your organisation? [Contact
+us by clicking the "Book a Demo" button on the Syntasso
+website](https://syntasso.io). 
 
-With Kratix and Promises, Platform Engineers can offer anything-as-a-service, consistently enforce business rules, and streamline maintenance by managing all resources collectively, like a fleet.
-:::tip
-Learn more about platform team cognitive load from [Paula Kennedy at PlatformCon](https://www.youtube.com/watch?v=zfKwxL9KZ9I)
 :::
 
-### Domain experts & Specialists (Promise Authors) 
-Many organisations have teams that specialise in specific domains, such as security, compliance, databases, or networking. These teams are responsible for ensuring that application teams follow the relevant business rules and policies.
+## How Kratix Works: Promises
+
+At Kratix's core are **Promises**—contracts between your platform and your
+application teams. Each Promise defines exactly what your platform provides
+as-a-service and includes:
+
+- **Self-Service APIs**: Clear, frictionless interfaces enabling immediate and
+autonomous resource provisioning.
+- **Embedded Workflows**: Automated processes embedded directly within each
+Promise, ensuring consistent compliance and security.
+- **Fleet Management**: Built-in reconciliation ensures resources remain
+up-to-date, simplifying management at scale.
+
+:::info
+Ready to get started? Follow our [Quick Start](01-quick-start.md) to jump right in.
+:::
+
+## Example: Self-Service Database Promise
```

Comment:
I wonder if this should be a different page? feels like a enough scrolling for one page 💭